modules/upgrade/pages/server-minor-upgrade-uyuni.adoc

```diff
@@ -35,26 +37,23 @@ zypper ref
 ----
 +
 
-. List available patches:
+// Zypper list-patches or patch doesn't update Uyuni
+. Update new packages: (Repeat this if prompted by zypper)
 +
 
 ----
-zypper list-patches
+zypper up
 ----
 +
 
-. Apply all available patches:
+. If zypper reports that the Uyuni package won't be upgraded, run that manually:
```

Comment:
"If **Zypper** reports that the Uyuni package will not be upgraded, run the following command manually:"